lib/urlwatch/handler.py

```diff
@@ -37,8 +37,15 @@
     import sha
     have_hashlib = False
 
+# Python 2 and 3 compatible string check
+try:
+    basestring
+except NameError:
+    basestring = str
+
 import subprocess
 import email.utils
+from urllib import urlencode
```

Comment:
Please use `import urllib` here and `urllib.urlencode` below to be consistent with the other imports here.